src/partials/reviews.html

```diff
@@ -0,0 +1,85 @@
+<div class="reviews__section">
+    <div class="reviews__container">
+
+        <div class="reviews__item-wrapper">
+
+            <div class="reviews__client is-faded is-visible" js-client-1>
```

Comment:
Nie lepiej zmienić .reviews__client na .client i dalej analogicznie?
To tylko sugestia (można zmienić w kolejnym branch)